src/App.css

```diff
@@ -1,3 +1,269 @@
 .App {
+  background-color: #fafafa;
+  font-family: Montserrat, sans-serif;
+}
+
```

Comment:
- [ ] There are lots of repeated properties in your CSS file.  Please don't repeat yourself (DRY).

## Explanation

This is an example of multiple CSS rules repeated:

![image](https://user-images.githubusercontent.com/57421823/166853278-5a936d40-9ded-41ae-9f8e-a4e97666b413.png)

We need to avoid this because it is a beginner's mistake.

## Hints:

- Go [here](https://speckyboy.com/good-bad-css-practices/) for support.
- 
![image](https://user-images.githubusercontent.com/57421823/166853364-005802ca-e681-443b-bed8-2d4a6818ab4f.png)

- 
![image](https://user-images.githubusercontent.com/57421823/166853376-e391ee87-04e9-4b20-b3e5-8858464e578d.png)

That way, your CSS will be written like a pro 💪

Reply:
Okay, thank you for the advice